BEIMA.Backend.FT/BeimaClient.cs

```diff
@@ -118,6 +119,55 @@ public async Task<HttpResponseMessage> SendRequest(string route, HttpVerb verb,
             return response;
         }
 
+        /// <summary>
+        /// Sends an multipart form data http request to the backend server and returns the http response.
+        /// </summary>
+        /// <param name="route">The url route excluding the base address.</param>
+        /// <param name="data">The object to send with the request.</param>
+        /// <param name="files">List of files to send with the request</param>
+        /// <returns>The http response message of the request.</returns>
+        /// <exception cref="HttpRequestException"></exception>
+        public async Task<HttpResponseMessage> SendMultiPartRequest(string route, Object data, FormFileCollection? files = null)
```

Comment:
We should include the option for adding a query string, just in case we end up using it for something.